app/(dashboard)/student/courses/[courseId]/learn/[sectionId]/[lectureId]/page.tsx

```
@@ -172,9 +172,9 @@ export default function LecturePage() {
   const params = useParams();
   const { courseId, sectionId, lectureId } = params;
 
-  const [course, setCourse] = useState<any>(null);
-  const [currentSection, setCurrentSection] = useState<any>(null);
-  const [currentLecture, setCurrentLecture] = useState<any>(null);
+  const [course, setCourse] = useState<any>(mockCourseContent);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)